cinema_service/cinema/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
It seems like you haven't written any tests yet. Consider adding test cases to verify the functionality of your models, views, and serializers. This will help ensure that your application behaves as expected.